build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -7,12 +7,8 @@ parameters:
 # The "primary" build arch is the one that the nuspec gets its winmd, pri, and other neutral files from
   primaryBuildArch: x86
   buildFlavor: Release
-  useReleaseTag: 'false'
   prereleaseVersionTag: 'prerelease'
```

Comment:
Why's major/minor/patch removed but tag still present?

Or that too is going just not there yet?

Reply:
The tag is present as a parameter because we have two different entries to the Foundation build. One internally and one in the 'ms' org azdo.